### ISSUE TITLE
GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.Flatpak.SystemHelper was not provided by any .service files

### ISSUE THREAD

**Issue by djspacewhale** [flatpak/flatpak#1816]
## Linux distribution and version

Fedora 28

## Flatpak version

Flatpak 0.11.8.3

## Description of the problem

```
flatpak update -v
F: Opening user flatpak installation at path /home/djspacewhale/.local/share/flatpak
F: Opening system flatpak installation at path /var/lib/flatpak
Looking for updates...
F: Fetching summary file for remote ‘flathub’
F: Updating dependent runtime runtime/org.gnome.Platform/x86_64/3.26
F: Transaction: update flathub:runtime/org.gnome.Platform/x86_64/3.26[$old]
F: No installations directory in /etc/flatpak/installations.d. Skipping
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.VAAPI.Intel/x86_64/1.6[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.ffmpeg/x86_64/1.6[*]
F: Transaction: install/update flathub:runtime/org.gnome.Platform.Locale/x86_64/3.26[/en]
F: Transaction: update flathub:app/com.github.lainsce.coin/x86_64/stable[$old]
F: Transaction: install/update flathub:runtime/com.github.lainsce.coin.Locale/x86_64/stable[*]
F: Transaction: update flathub:app/com.github.z.Cumulonimbus/x86_64/stable[$old]
F: Updating dependent runtime runtime/org.freedesktop.Sdk/x86_64/1.6
F: Transaction: update flathub:runtime/org.freedesktop.Sdk/x86_64/1.6[$old]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.VAAPI.Intel/x86_64/1.6[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.ffmpeg/x86_64/1.6[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Sdk.Locale/x86_64/1.6[/en]
F: Transaction: update flathub:app/com.google.AndroidStudio/x86_64/stable[$old]
F: Transaction: install/update flathub:runtime/com.google.AndroidStudio.Locale/x86_64/stable[*]
F: Updating dependent runtime runtime/org.freedesktop.Platform/x86_64/1.6
F: Transaction: update flathub:runtime/org.freedesktop.Platform/x86_64/1.6[$old]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.VAAPI.Intel/x86_64/1.6[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.ffmpeg/x86_64/1.6[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.Locale/x86_64/1.6[/en]
F: Transaction: update flathub:app/com.googleplaymusicdesktopplayer.GPMDP/x86_64/stable[$old]
F: Updating dependent runtime runtime/org.gnome.Platform/x86_64/3.28
F: Transaction: update flathub:runtime/org.gnome.Platform/x86_64/3.28[$old]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.VAAPI.Intel/x86_64/1.6[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.ffmpeg/x86_64/1.6[*]
F: Transaction: install/update flathub:runtime/org.gnome.Platform.Locale/x86_64/3.28[/en]
F: Transaction: update flathub:app/com.jagex.RuneScape/x86_64/stable[$old]
F: Updating dependent runtime runtime/org.kde.Platform/x86_64/5.9
F: Transaction: update flathub:runtime/org.kde.Platform/x86_64/5.9[$old]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.VAAPI.Intel/x86_64/1.6[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.ffmpeg/x86_64/1.6[*]
F: Transaction: install/update flathub:runtime/org.kde.Platform.Locale/x86_64/5.9[/en]
F: Transaction: update flathub:app/com.obsproject.Studio/x86_64/stable[$old]
F: Transaction: install/update flathub:runtime/com.obsproject.Studio.Locale/x86_64/stable[*]
F: Transaction: update flathub:app/com.realm667.WolfenDoom_Blade_of_Agony/x86_64/stable[$old]
F: Transaction: update flathub:app/com.skype.Client/x86_64/stable[$old]
F: Transaction: update flathub:app/com.snes9x.Snes9x/x86_64/stable[$old]
F: Transaction: install/update flathub:runtime/com.snes9x.Snes9x.Locale/x86_64/stable[*]
F: Transaction: update flathub:app/com.spotify.Client/x86_64/stable[$old]
F: Transaction: update flathub:app/com.sublimetext.three/x86_64/stable[$old]
F: Transaction: update flathub:app/com.valvesoftware.Steam/x86_64/stable[$old]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.Compat32/x86_64/1.6[*]
F: Transaction: update flathub:app/com.vinszent.GnomeTwitch/x86_64/stable[$old]
F: Transaction: install/update flathub:runtime/com.vinszent.GnomeTwitch.Locale/x86_64/stable[*]
F: Transaction: update flathub:app/im.gitter.Gitter/x86_64/stable[$old]
F: Transaction: update flathub:app/io.atom.Atom/x86_64/stable[$old]
F: Transaction: update flathub:app/io.mgba.mGBA/x86_64/stable[$old]
F: Transaction: update flathub:app/net.openra.OpenRA/x86_64/stable[$old]
F: Updating dependent runtime runtime/org.freedesktop.Platform/i386/1.6
F: Transaction: update flathub:runtime/org.freedesktop.Platform/i386/1.6[$old]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.VAAPI.Intel/i386/1.6[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.ffmpeg/i386/1.6[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.Locale/i386/1.6[/en]
F: Transaction: update flathub:app/net.pcsx2.PCSX2/i386/stable[$old]
F: Transaction: install/update flathub:runtime/net.pcsx2.PCSX2.Locale/i386/stable[*]
F: Transaction: update flathub:app/org.DolphinEmu.dolphin-emu/x86_64/stable[$old]
F: Transaction: install/update flathub:runtime/org.DolphinEmu.dolphin_emu.Locale/x86_64/stable[*]
F: Transaction: update flathub:app/org.ethereum.Mist/x86_64/stable[$old]
F: Transaction: update flathub:app/org.geogebra.GeoGebra/x86_64/stable[$old]
F: Transaction: update flathub:app/org.gnome.gitg/x86_64/stable[$old]
F: Transaction: install/update flathub:runtime/org.gnome.gitg.Locale/x86_64/stable[*]
F: Transaction: update flathub:app/org.godotengine.Godot/x86_64/stable[$old]
F: Transaction: update flathub:app/org.musescore.MuseScore/x86_64/stable[$old]
F: Transaction: update flathub:app/org.openmw.OpenMW/x86_64/stable[$old]
F: Transaction: update flathub:app/org.openshot.OpenShot/x86_64/stable[$old]
F: Transaction: update flathub:app/org.ppsspp.PPSSPP/x86_64/stable[$old]
F: Transaction: update flathub:app/org.remmina.Remmina/x86_64/stable[$old]
F: Transaction: install/update flathub:runtime/org.remmina.Remmina.Locale/x86_64/stable[*]
F: Transaction: update flathub:app/org.telegram.desktop/x86_64/stable[$old]
F: Transaction: update flathub:app/org.zotero.Zotero/x86_64/stable[$old]
F: Transaction: update flathub:runtime/com.github.lainsce.coin.Locale/x86_64/stable[$old]
F: Transaction: update flathub:runtime/com.google.AndroidStudio.Locale/x86_64/stable[$old]
F: Transaction: update flathub:runtime/com.obsproject.Studio.Locale/x86_64/stable[$old]
F: Transaction: update flathub:runtime/com.snes9x.Snes9x.Locale/x86_64/stable[$old]
F: Transaction: update flathub:runtime/com.vinszent.GnomeTwitch.Locale/x86_64/stable[$old]
F: Transaction: update flathub:runtime/net.pcsx2.PCSX2.Locale/i386/stable[$old]
F: Transaction: update flathub:runtime/org.DolphinEmu.dolphin_emu.Locale/x86_64/stable[$old]
F: Transaction: update flathub:runtime/org.freedesktop.Platform.Compat32/x86_64/1.6[$old]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.VAAPI.Intel/x86_64/1.6[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.ffmpeg/x86_64/1.6[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.Locale/x86_64/1.6[/en]
F: Transaction: update flathub:runtime/org.freedesktop.Platform.Locale/i386/1.6[$old]
F: Transaction: update flathub:runtime/org.freedesktop.Platform.Locale/x86_64/1.6[$old]
F: Transaction: update flathub:runtime/org.freedesktop.Platform.VAAPI.Intel/i386/1.6[$old]
F: Transaction: update flathub:runtime/org.freedesktop.Platform.VAAPI.Intel/x86_64/1.6[$old]
F: Transaction: update flathub:runtime/org.freedesktop.Platform.ffmpeg/i386/1.6[$old]
F: Transaction: update flathub:runtime/org.freedesktop.Platform.ffmpeg/x86_64/1.6[$old]
F: Transaction: update flathub:runtime/org.freedesktop.Platform/i386/1.6[$old]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.VAAPI.Intel/i386/1.6[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.ffmpeg/i386/1.6[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.Locale/i386/1.6[/en]
F: Transaction: update flathub:runtime/org.freedesktop.Platform/x86_64/1.6[$old]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.VAAPI.Intel/x86_64/1.6[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.ffmpeg/x86_64/1.6[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.Locale/x86_64/1.6[/en]
F: Transaction: update flathub:runtime/org.freedesktop.Sdk.Locale/x86_64/1.6[$old]
F: Transaction: update flathub:runtime/org.freedesktop.Sdk/x86_64/1.6[$old]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.VAAPI.Intel/x86_64/1.6[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.ffmpeg/x86_64/1.6[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Sdk.Locale/x86_64/1.6[/en]
F: Transaction: update flathub:runtime/org.gnome.Platform.Locale/x86_64/3.26[$old]
F: Transaction: update flathub:runtime/org.gnome.Platform.Locale/x86_64/3.28[$old]
F: Transaction: update flathub:runtime/org.gnome.Platform/x86_64/3.26[$old]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.VAAPI.Intel/x86_64/1.6[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.ffmpeg/x86_64/1.6[*]
F: Transaction: install/update flathub:runtime/org.gnome.Platform.Locale/x86_64/3.26[/en]
F: Transaction: update flathub:runtime/org.gnome.Platform/x86_64/3.28[$old]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.VAAPI.Intel/x86_64/1.6[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.ffmpeg/x86_64/1.6[*]
F: Transaction: install/update flathub:runtime/org.gnome.Platform.Locale/x86_64/3.28[/en]
F: Transaction: update flathub:runtime/org.gnome.gitg.Locale/x86_64/stable[$old]
F: Transaction: update flathub:runtime/org.kde.Platform.Locale/x86_64/5.11[$old]
F: Transaction: update flathub:runtime/org.kde.Platform.Locale/x86_64/5.9[$old]
F: Transaction: update flathub:runtime/org.kde.Platform/x86_64/5.11[$old]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.VAAPI.Intel/x86_64/1.6[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.ffmpeg/x86_64/1.6[*]
F: Transaction: install/update flathub:runtime/org.kde.Platform.Locale/x86_64/5.11[/en]
F: Transaction: update flathub:runtime/org.kde.Platform/x86_64/5.9[$old]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.VAAPI.Intel/x86_64/1.6[*]
F: Transaction: install/update flathub:runtime/org.freedesktop.Platform.ffmpeg/x86_64/1.6[*]
F: Transaction: install/update flathub:runtime/org.kde.Platform.Locale/x86_64/5.9[/en]
F: Transaction: update flathub:runtime/org.remmina.Remmina.Locale/x86_64/stable[$old]
F: Updating remote metadata for flathub
Updating: com.snes9x.Snes9x/x86_64/stable from flathub
[####################] 1 delta parts, 2 loose fetched; 946 KiB transferred in 1 F: Calling system helper: Deploy

Error: Failed to update com.snes9x.Snes9x/x86_64/stable: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.Flatpak.SystemHelper was not provided by any .service files
Updating: com.snes9x.Snes9x.Locale/x86_64/stable from flathub
[####################] 1 delta parts, 1 loose fetched; 20 KiB transferred in 0 sF: Calling system helper: Deploy

Warning: Failed to update com.snes9x.Snes9x.Locale/x86_64/stable: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.Flatpak.SystemHelper was not provided by any .service files
error: There were one or more errors
```
## Steps to reproduce

This happens for me whenever I attempt to install a flatpak package or update - here's a similar error attempting to install discord:
```
flatpak install flathub com.discordapp.Discord
Installing: com.discordapp.Discord/x86_64/stable from flathub
[####################] Downloading files: 1/1 71.5 MB (35.8 MB/s)         
error: Failed to install com.discordapp.Discord/x86_64/stable: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.Flatpak.SystemHelper was not provided by any .service files
```

Note that I've also attempted reinstalling flatpak and flatpak-* then rebooting and that did not fix the error

**Comment by alexlarsson** [comment]
I believe this is fixed by #1873

**Comment by rujiel** [comment]
> I believe this is fixed by #1873

I'm having this issue on linux mint 21, and I'm not clear how that issue is related

**Comment by rujiel** [comment]
This error also appears in the case that you try to install a flatpak without flatpak itself being installed
